addie/menu/event_handler.py

```diff
@@ -54,7 +54,7 @@ def help_about_clicked(main_window):
 
 
 def activate_reduction_tabs(main_window):
-    if main_window.post_processing == 'idl':
+    if main_window.post_processing in ("idl", "idl_dev"):
```

Comment:
Maybe we put this as a variable in `addie.main` that we can import?
Like `addie.main.IDL_MODES == ('idl', 'idl_dev')?

Then we can import and use it here and below in `addie.menu.file.settings.advanced_file_window`